README.md

```diff
@@ -136,8 +136,8 @@ packages:
 ## Package Maintenance
 The Fivetran team maintaining this package _only_ maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/reddit_ads/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_reddit_ads/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
 
-## Opinionated Decisions
-In creating this package, which is meant for a wide range of use cases, we had to take opinionated stances on a few different questions we came across during development. We've consolidated significant choices we made in the [DECISIONLOG.md](https://github.com/fivetran/dbt_reddit_ads/blob/main/DECISIONLOG.md), and will continue to update as the package evolves. We are always open to and encourage feedback on these choices, and the package in general.
+## Opinionated Decisions 
+In creating this package, which is meant for a wide range of use cases, we had to take opinionated stances on a few different questions we came across during development. We've consolidated significant choices we made in the [DECISIONLOG.md](https://github.com/fivetran/dbt_reddit_ads/blob/main/DECISIONLOG.md), and will continue to update as the package evolves. We are always open to and encourage feedback on these choices, and the package in general. 
```

Comment:
Same comment as before, is there a need for the decisionlog in this package?

Reply:
It looks like we may want to include a blurb about the default behavior of the url model. See how we phrased this for [twitter_ads](https://github.com/fivetran/dbt_twitter/blob/main/DECISIONLOG.md)

Reply:
Do we need it? Reddit ads URLs are pretty straightforward from my understanding, with one url per ad. The twitter blurb is regarding how there can be multiple links per tweet, so requires an index

Reply:
Makes sense! Let's remove.